packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -191,6 +192,18 @@ function useLayoutEffect(
   });
 }
 
+function useInsertionEffect(
```

Comment:
Didn't we decide to drop the `ion` and just do `useInsertEffect`?

Reply:
Yes, and then no. `useInsertEffect` sounds too much like the thing you're inserting is an effect, which I think we noticed when we started talking about it again and thought of it as a verb first (which is probably the natural reading of it).